packages/create-wave-app/src/questions/index.ts

```diff
@@ -72,7 +72,7 @@ export const generateQuestions = [
       {
         name: "pnpm",
         value: "pnpm",
-        disabled: !isPnpmInstalled,
+        disabled: !isPnpmInstalled && "not installed",
```

Comment:
where does the "not installed" string come from?

Reply:
it's what is displayed in the command line if no pnpm is detected. it will disable the option and say in brackets (not installed)